index.html

```diff
@@ -0,0 +1,132 @@
+                <h4 class="navbar-brand" href="#">Arinze's <span>Design</span></h4>
+                <button class="navbar-toggler" type="button" data-toggle="collapse"
+                    data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav ml-auto">
+                        <li class="nav-item"><a class="nav-link active-link" href="#">Home</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#">Portfolio</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#about">About Me</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#social-media">Contact</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <main>
+            <section id="homepage">
+                <div class="home-text">
+                    <img src="images/picture.jpg" alt="personal picture">
+                    <p class="name">Hi! I'm <span>Arinze I.C</span></p>
+                    <p class="desc">Front-end web developer</p>
+                    <a href="#">My Work</a>
+                </div>
+            </section>
+    </div>
+    <section id="social-media">
+        <h3>Let's connect:</h3>
+        <div>
+            <a href="https://www.instagram.com/ari.n.ze/" target="_blank"><i class="fab fa-instagram"></i></a>
+            <a href="https://www.linkedin.com/in/arinze-ihemedu-14b738148/" target="_blank"><i
+                    class="fab fa-linkedin-in"></i></a>
+            <a href="https://www.quora.com/profile/Arinze-Ihemedu" target="_blank"><i class="fab fa-quora"></i></a>
+            <a href="https://github.com/Arinzemike1" target="_blank"><i class="fab fa-github"></i></a>
+        </div>
+    </section>
+    <section id="about">
+        <div class="container">
+            <h1>About Me</h1>
+            <div class="row">
+                <div class="col-md-6 text-center about-col">
+                    <div class="profile">
+                        <img src="images/picpic.jpg" class="user" alt="about image">
+                        <p>Hi there! I'm Arinze Ihemedu, a graduate of applied informatics and system programming from
+                            Vitebsk State University. I've got some basic understanding of c++ programming language but
+                            here I am doing what I love most which is web development. My stack: HTML5, CSS3/Bootstrap,
+                            Javascript/Jquery.</p>
+                    </div>
+                </div>
+                <div class="col-md-6 skills-bar">
+                    <p>HTML</p>
+                    <div class="progress">
```

Comment:
Use semantic html here and in all occurrences. Refer to https://www.w3schools.com/tags/tag_progress.asp